docs/application/dotnet/guides/nui/styling-controls-with-JSON.md

```diff
@@ -230,6 +230,36 @@ You can implement transitions in two ways:
     ]
     ```
 
+## Apply JSON style
+
+Style can be applied at runtime using the **StyleManager** API, as shown in the following example code:
+
+```
+using Tizen.NUI.Components;
+using Tizen.NUI.BaseComponents;
+
+//Load json theme
+Tizen.NUI.StyleManager.Get().ApplyTheme(DirectoryInfo.Resource + "/styles/theme.json");
+
+//Create button object
+Button testButton = new Button();
+testButton.SetStyleName("CustomButton");
+```
+
+## Handle theme change
+
+It is possible to observe whether the used theme was changed using the `Tizen.NUI.StyleManager.Get().StyleChanged` event. If there is a need to perform the action when the new theme is loaded, you can use the following example code:
```

Comment:
How about below?

To get a notification when the used theme is changed, you can register an event handler for the `StyleChanged` event  as the following code:

Reply:
done

Reply:
@mijin85cho  the rephrase given by you, can we slightly alter this? to give the sentence better comprehension. 

```suggestion
To get a notification when the current theme is changed, you can register an event handler for the `StyleChanged` event as follows:

Reply:
@nawab-samsung I think your suggestion is acceptable.